src/app/tab1/tab1.page.html

```diff
@@ -9,19 +29,23 @@
 <ion-content>
   <ion-list lines="none">
     <ion-list-header>
-      <ion-label>trending tv shows</ion-label>
+      <ion-label><h1>Trending tv shows</h1></ion-label>
     </ion-list-header>
     <ion-item ion-item *ngFor="let tvshow of tvshows" >
-      <ion-icon slot="start" color="medium" name="book"></ion-icon>
+      <ion-thumbnail class="thumbnail" slot="start">
+        <ion-img class="thumbnail" [src]="'https://image.tmdb.org/t/p/w200/'+tvshow.poster_path"></ion-img>
+      </ion-thumbnail>
       <ion-label>
         <h2>{{tvshow.name}}</h2>
-
-        <div ion-item *ngFor="let genre of tvshow.genres" >
-          <p>{{genre.name}}</p>
+        <h4>Release on: {{tvshow.first_air_date}}</h4>
+        <div class="genres">
+          <div ion-item *ngFor="let genre of tvshow.genres" >
+            <ion-badge class="genre" color="dark">{{genre.name}}</ion-badge>
+          </div>
         </div>
-
+        <p class="overview">{{tvshow.overview}}</p>
```

Comment:
```suggestion
        <p class="overview">{{ tvshow.overview }}</p>
```

Spaties tussen template tekens